bin/init_repo.sh

```diff
@@ -32,6 +32,8 @@ templates=(
     "templates/continous-delivery.yml.tmpl:.github/workflows/continous-delivery.yml"
     "templates/continous-integration.yml.tmpl:.github/workflows/continous-integration.yml"
     "templates/github-page-build-and-deploy.yml.tmpl:.github/workflows/github-page-build-and-deploy.yml"
+    "templates/source_index.rst.tmpl:sphinx/source/index.rst"
+    "templates/userguide_index.rst.tmpl:sphinx/source/userguide/index.rst"
```

Comment:
These two sphinx files should have been parametrized since the beginning since they include reference to the package name and a few other init parameters